### PR TITLE
calendar: Bind Org functions in calendar keymap

### PR DESCRIPTION
**Pull request by condy0919**
@Ambrevar

### CODE REVIEW

File: modes/calendar/evil-collection-calendar.el [hunk 1]
```diff
@@ -99,7 +134,11 @@
     ;; quit
     "q" 'calendar-exit
     "ZQ" 'evil-quit
-    "ZZ" 'calendar-exit))
+    "ZZ" 'calendar-exit)
+
+  (defvar evil-collection-setup-calendar-org-bindings)
```
Comment (Ambrevar) [inline modes/calendar/evil-collection-calendar.el:139]: Maybe move this defvar at the top-level with a docstring?

**Comment by Ambrevar** [comment]
Looks good to me.
@jojojames Thoughts on this?

File: modes/calendar/evil-collection-calendar.el (hunk 1 above)
Comment (condy0919) [reply modes/calendar/evil-collection-calendar.el:139]: The `defvar` is used to "declare" `evil-collection-setup-calendar-org-bindings` which is defined in `evil-collection.el`.

`evil-collection-ivy.el` does the same.

**Comment by condy0919** [comment]
Passed the CI check :tada:

**Comment by condy0919** [comment]
The customizable variable is named to `evil-collection-calendar-want-org-bindings`

**Comment by condy0919** [comment]
@jojojames Thoughts?

**Comment by jojojames** [comment]
Looks good thanks!

**Pull request merged**